From 6732170a5e2b4b6d42c954e4edcfd4d6dc269fa4 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Mon, 18 Apr 2016 03:32:25 -0400
Subject: [PATCH 12/62] use png icon to replace svg icon

WRLinux does not support svg display very well,
so we use png to workaround.

Upstream-Status: Inappropriate [wrlinux specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 widgets/src/DiskOverview.c  | 2 +-
 widgets/src/SpokeSelector.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/widgets/src/DiskOverview.c b/widgets/src/DiskOverview.c
index c716be8..d888141 100644
--- a/widgets/src/DiskOverview.c
+++ b/widgets/src/DiskOverview.c
@@ -275,7 +275,7 @@ static void set_icon(AnacondaDiskOverview *widget, const char *icon_name) {
             return;
         }
 
-        file = g_strdup_printf("%s/pixmaps/anaconda-selected-icon.svg", anaconda_get_widgets_datadir());
+        file = g_strdup_printf("%s/pixmaps/anaconda-selected-icon.png", anaconda_get_widgets_datadir());
         emblem_icon = g_icon_new_for_string(file, &err);
         g_free(file);
         if (!emblem_icon) {
diff --git a/widgets/src/SpokeSelector.c b/widgets/src/SpokeSelector.c
index 445f9c2..505f836 100644
--- a/widgets/src/SpokeSelector.c
+++ b/widgets/src/SpokeSelector.c
@@ -203,7 +203,7 @@ static void set_icon(AnacondaSpokeSelector *widget, const char *icon_name) {
             return;
         }
 
-        file = g_strdup_printf("%s/pixmaps/dialog-warning-symbolic.svg", anaconda_get_widgets_datadir());
+        file = g_strdup_printf("%s/pixmaps/dialog-warning-symbolic.png", anaconda_get_widgets_datadir());
         emblem_icon = g_icon_new_for_string(file, &err);
         g_free(file);
         if (!emblem_icon) {
-- 
2.7.4

